modules/apps/object/object-api/src/main/java/com/liferay/object/field/attachment/AttachmentManager.java

```diff
@@ -18,17 +18,6 @@
  */
 public interface AttachmentManager {
 
-	public FileEntry addFileEntry(
```

Comment:
Hi @pedro-oliveira446 , because this is a public method of an API, can we mark the method as `deprecated` instead of removing it?

Reply:
Done!